en/controllers.rst

```diff
@@ -277,6 +277,47 @@ For example::
 
 This would render **plugins/Users/templates/UserDetails/custom_file.php**
 
+.. _controller-viewclasses:
+
+Content Type Negotiation
+========================
+
+Controllers can define a list of view classes they support. After the
+controller's action is complete CakePHP will use the view list to perform
+content-type negotiation. This enables your application to re-use the same
+controller action to render an HTML view or render a JSON or XML response. To
+define the list of supported view classes for a controller is done with the
+``viewClasses()`` method::
+
+    namespace App\Controller;
+
+    use Cake\View\JsonView;
+    use Cake\View\XmlView;
+
+    class PostsController extends AppController
+    {
+        public function viewClasses()
+        {
+            return [JsonView::class, XmlView::class];
+        }
+    }
+
+You do not need to include the basic ``View`` class in your view class list as
+it is used as a fallback when no other view can be selected based on the
```

Comment:
```suggestion
The basic ``View`` class is automatically used as a fallback
when no other view can be selected based on the
```
Seems clearer to state this simply. Not sure where to explain it doesn't need to be listed either.